code/game/objects/items/contraband.dm

```diff
@@ -237,6 +234,59 @@
 	to_chat(user, "You unwrap the package.")
 	qdel(src)
 
+/obj/item/mechasalvage
+	name = "mystery mech salvage"
+	desc = "An oil and rust stained 'box'."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "deliverycrate5"
+	item_state = "table_parts"
+	w_class = ITEMSIZE_HUGE
+
+/obj/item/mechasalvage/attack_self(mob/user)
+	. = ..()
+	if(.)
+		return
+	var/loot = pick(/obj/item/mecha_parts/chassis/phazon,
+		/obj/item/mecha_parts/part/phazon_head,
+		/obj/item/mecha_parts/part/phazon_left_arm,
+		/obj/item/mecha_parts/part/phazon_left_leg,
+		/obj/item/mecha_parts/part/phazon_right_arm,
+		/obj/item/mecha_parts/part/phazon_right_leg,
+		/obj/item/mecha_parts/part/phazon_torso,
```

Comment:
the phazon is currently intended to be admin spawn only
afaik

Reply:
It also can (rarely) be rolled out of enough Bulk Contraband Crates, so there is at least justification for players being able to get it, though this significantly takes down the average price to roll one.

Reply:
It really doesn't reduce the price. If you actively try rolling it now. you **might** end up rolling a nice jackpot and get it easier, but on average, it still costs a shitton of money, and you're left with a fuckton more expensive mechparts that you may not have the materials to finish up. 

Also as is, I had to fix up certain things with the incomplete gatcha, like giving station the ability to manufacture the boards for it since, y'know, nobody bothered to remove the parts wholesale or not, so I made the executive decision of making it possible to acquire. Additionally, you need a head of staff to approve the use of the hand teleporter in its construction, or commit a very huge crime, acquiring command equipment without authorization, and something so unique at that. It's a "this is a nice thing to do" sort of thing, but not something you should do every shift.